elastic/apm-aws-lambda
# Create proxy transasction if not reported by agent

## Motivation

Create proxy transactions if agent is not able to report the transaction due to unexpected errors like `OutOfMemory`.

## Related issues

Part of #315 

## Testing

1. Create a lambda function that causes an OOM and run it a few times.
2. Observe on elastic to assert that a transaction for the function invocation is reported.
